.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: Report a Gson bug.
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+# Gson version
+<!-- Gson version you are using, for example '2.8.8' -->
+
+
+# Java / Android version
+<!-- Version of the Java or Android platform on which the bug occurred -->
+
+
+# Used tools
+<!-- List relevant build tools here which might affect Gson -->
+- [ ] ProGuard
```

Comment:
Perhaps Maven and Gradle too? I wonder if it is worth asking for versions as well.

Reply:
Sounds like a good idea; Maven and Gradle themselves probably in most cases 'only' fetch the dependencies and build the project, but there could be cases where the build tool itself makes a difference.

What do you think about this change?
```markdown
# Used tools
<!-- List relevant build tools and plugins with version number here which might affect Gson -->
- [ ] Maven; version: 
- [ ] Gradle; version: 
- [ ] ProGuard (attach the configuration file please); version: 
- [ ] ...
```

Reply:
That looks good to me.